core/src/main/java/org/geysermc/geyser/translator/protocol/bedrock/BedrockCommandRequestTranslator.java

```diff
@@ -39,15 +39,17 @@ public class BedrockCommandRequestTranslator extends PacketTranslator<CommandReq
     @Override
     public void translate(GeyserSession session, CommandRequestPacket packet) {
         String command = MessageTranslator.convertToPlainText(packet.getCommand());
+        handleCommand(session, MessageTranslator.normalizeSpace(command).substring(1));
+    }
+
+    public static void handleCommand(GeyserSession session, String command) {
```

Comment:
This can be `static void` instead of `public`

Reply:
resolved here - https://github.com/GeyserMC/Geyser/pull/4703/commits/04d902466401400587260ab0c182ccfab7c65614